scripts/exgdas_global_atmos_analysis_run.sh

```diff
@@ -83,6 +86,24 @@ done
 # change to (or make optional) for cube sphere history later
 $NLN ${COMIN_GES}/RESTART $DATA/bkg
 
+
+# optionally link ensemble backgrounds to $DATA
+if [ $DOHYBVAR = "YES" ]; then
```

Comment:
So I guess this is a temporary thing since R2D2 can't handle ensemble forecasts...

Reply:
Yes, this section of scripting could be replaced by R2D2 at some point.  We should sit down sometime with EIB and DA staff to discuss the use of R2D2 in operations.